blivet/formats/fs.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, **kwargs):
             :keyword mountpoint: the filesystem's planned mountpoint
             :keyword label: the filesystem label
             :keyword uuid: the filesystem UUID
+            :keyword new_uuid: new UUID to set for this filesystem
```

Comment:
I don't understand this. We know if the file system exists (in the system) or not (just in the model). If the UUID is set before it even exists, it means that it is requested to be set on creation, doesn't it? And other parts of blivet shouldn't care about these things, they just need to know the UUID of the file system when everything is created (for ``/etc/fstab``,...)